pycbc/events/stat.py

```diff
@@ -317,19 +317,24 @@ def get_hist(self, ifos=None):
 
         for ifo in self.hist_ifos:
             self.weights[ifo] = histfile[ifo]['weights'][:]
-
             param = histfile[ifo]['param_bin'][:]
-            ncol = param.shape[1]
-            self.pdtype = [('c%s' % i, param.dtype) for i in range(ncol)]
-            self.param_bin[ifo] = numpy.zeros(len(self.weights[ifo]),
-                                              dtype=self.pdtype)
-            for i in range(ncol):
-                self.param_bin[ifo]['c%s' % i] = param[:, i]
-
-            lsort = self.param_bin[ifo].argsort()
-            self.param_bin[ifo] = self.param_bin[ifo][lsort]
-            self.weights[ifo] = self.weights[ifo][lsort]
-
+            if param.dtype == 'int8':
```

Comment:
I'm not sure this is right. I don't think `param.dtype` would be a string. 

I think this would be:

```if param.dtype == np.int8```

I don't know if there's a better way to check if the `dtype` is *any* integer-like type, but if we don't expect to use this code again, then I think just supporting the old file, and only the old file, is fine.

Reply:
I have no idea why this has had to be changed - I am sure that I originally tried ==numpy.int8 (as that makes more sense) but thought that that broke it and I had to use "int"

I'll sort it but think the lockdown may be getting to me